compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -662,7 +674,8 @@ static bool willNotInlineCompareAndSwapNative(TR::Node *node,
 #endif
    }
 
-//some recognized methods might be transformed into very simple hardcoded assembly sequence which doesn't need register spills as a real call does
+// Some recognized methods are transformed into very simple hardcoded
+// assembly sequences which don't need register spills as real calls do
```

Comment:
Not sure how this comment relates to the method?

Reply:
As far as I can see the comment is accurate. What sort of changes did you have in mind?

Reply:
I think a proper doxygen stating this method return false if the code generator has special handling for a given method that will result in a simple inline assembly sequence or similar. The current comment just seems a bit out of context - albeit related.

Reply:
Ah, ok. I'll do that.